README.md

```diff
@@ -50,7 +50,7 @@ Applications
 
 - Fork and clone the repo: `git clone https://github.com/<username>/noobaa-operator`
 - Use minikube: `minikube start`
-- Use you package manager to install `go`, `operator-sdk` and `python3`.
+- Use your package manager to install `go`, `operator-sdk` and `python3`.
```

Comment:
Perhaps a link to operator-sdk repo on how to do it, as clearly it's not just the regular package manager:
```
[ykaul@ykaul glusterfs]$ sudo dnf install operatror-sdk
[sudo] password for ykaul: 
RHIT IT Legacy User Configs                                                                                                                                                                                   346  B/s | 224  B     00:00    
Errors during downloading metadata for repository 'idm-legacy-configs':
  - Status code: 404 for http://sat6capsule.corp.redhat.com/pulp/repos/rhit/Library/custom/iam-idm/idm-legacy-configs/repodata/repomd.xml (IP: 10.39.142.41)
Error: Failed to download metadata for repo 'idm-legacy-configs': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
Ignoring repositories: idm-legacy-configs
Last metadata expiration check: 2:52:33 ago on Mon 24 Feb 2020 07:23:27 IST.
No match for argument: operatror-sdk
Error: Unable to find a match: operatror-sdk
```